doc/data/messages/a/assert-on-string-literal/good.py

```diff
@@ -1,2 +1,4 @@
 def test():
-    # Nothing, as an assert of a string literal will always pass
+    actual = "test"
+    wanted = "Directly asserting a string literal will always pass"
+    assert actual == wanted
```

Comment:
Shall we leave the comment or put it in `details.rst`?

Reply:
I don't have strong feelings. This doesn't feel like it rises to a "gotcha" or "for further reading". But I needed to define the constants with something. But I get that this looks weird as is. Happy to go either way!

Reply:
In the original PR (https://github.com/PyCQA/pylint/pull/5960) Pierre was against silly assertions so that's why I added the comments. I think doing both is the best middle ground.

Reply:
I think I see, what do you think of this?

Reply:
Seems perfect! Only need it for the other message as well!